test/smoke/src/areas/positron/console/consoleInput.test.ts

```diff
@@ -81,10 +81,67 @@ cat(sprintf('Hello %s!\n', val))`;
 
 					await app.workbench.positronConsole.sendEnterKey();
 
-					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Hello John Doe!')) );
-				}).toPass({timeout: 60000});
+					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Hello John Doe!')));
+				}).toPass({ timeout: 60000 });
 
 			});
+
+			it('R - Can use `menu` to select alternatives [C684749]', async function () {
+				const app = this.app as Application;
+				const inputCode = `x <- menu(letters)`;
+
+				await expect(async () => {
+					await app.workbench.positronConsole.pasteCodeToConsole(inputCode);
+					await app.workbench.positronConsole.sendEnterKey();
+
+					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Selection:')));
+
+					// slight wait before starting to type
+					await app.code.wait(200);
+
+					await app.workbench.positronConsole.typeToConsole('1');
+					await app.workbench.positronConsole.sendEnterKey();
+
+					// slight wait before starting to type
+					await app.code.wait(200);
+
+					await app.workbench.positronConsole.typeToConsole('x');
+					await app.workbench.positronConsole.sendEnterKey();
+
+					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('[1] 1')));
+				}).toPass({ timeout: 60000 });
+			});
+
+			it("R - Esc only dismisses autocomplete not full text typed into console", async function () {
```

Comment:
@jonvanausdeln I think we are missing a testrail id here.

Reply:
Added. Sorry, not sure how I missed that one, I had it created.